ext/intl/locale/locale_methods.c

```diff
@@ -1619,3 +1619,19 @@ PHP_FUNCTION(locale_accept_from_http)
 	RETURN_STRINGL(resultLocale, len);
 }
 /* }}} */
+
+PHP_FUNCTION(locale_is_right_to_left)
+{
+	char *locale;
+	size_t locale_len;
+
+	ZEND_PARSE_PARAMETERS_START(1, 1)
+		Z_PARAM_STRING(locale, locale_len)
```

Comment:
Nit: Seems like this could be PATH_STRING because NULL bytes are treated as the end of the string.
Although it seems this is not done in this file.